base_unece/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_unece_code_list_read,Read access on unece.code.list to everybody,model_unece_code_list,,1,0,0,0
```

Comment:
there is a warning caused by missing group, no ?
Take inspiration on the ACL of res.country in the base module: they now have 3 read ACLs: base.group_user, base.group_portal and base.group_public

Reply:
ok @alexis-via, I will take that into consideration